controllers/packetcluster_controller.go

```diff
@@ -97,9 +97,12 @@ func (r *PacketClusterReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 		}
 	}()
 
-	clusterScope.PacketCluster.Status.Ready = true
```

Comment:
This part still isn't right. According to the flow, the cluster status is ready when the underlying infra is ready. In Packet's case, there is no underlying infra, so it is ready immediately.

The flow should be something like:

1. Cluster controller:
    1. Set up underlying cluster infra (we have none)
    1. mark cluster as ready
1. Machine controller: 
    1. Cluster not ready? Requeue
    1. Cluster ready? Create masters
    1. Cluster ready and have kubeconfig secret? Create workers 

The kubeconfig secret is created when the cluster is ready **and** the control plane endpoint is available.